src/lib/strategy-card/StrategyCardBorder.js

```diff
@@ -0,0 +1,31 @@
+const { Border } = require("../../wrapper/api");
```

Comment:
filename should follow strategy-card-border.js convention

Reply:
done